march_hardware/src/IMotionCube.cpp

```diff
@@ -76,41 +76,39 @@ void IMotionCube::validateMosiPDOs()
 void IMotionCube::writeInitialSettings(uint8 ecatCycleTime)
 {
   ROS_DEBUG("IMotionCube::writeInitialSettings");
-  bool success = true;
-  // sdo_bit32(slaveIndex, address, subindex, value);
 
   if (this->actuationMode == ActuationMode::unknown)
   {
     throw std::runtime_error("Cannot write initial settings to IMotionCube as it has actuation mode of unknown");
   }
 
```

Comment:
I like to see the addition of the register 6007. This enables the iMC with functionality to react to an etherCAT communication loss. 
```suggestion
sdo_bit8(slaveIndex, 0x6007, 0, 1);
```

Reply:
I will add and test this in another branch. This branch has been tested with exo and should be merged as quickly as possible.